builder/vsphere/common/step_reattach_cdrom_test.go

```diff
@@ -27,7 +27,7 @@ func TestStepReattachCDRom_Run(t *testing.T) {
 		errMessage     string
 	}{
 		{
-			name: "Successfully reattach CD-ROM device",
+			name: "Reattach 3 more CD-ROM devices",
```

Comment:
```suggestion
			name: "Successfully attach 3 additional CD-ROM devices",
```

Reply:
Done!

Reply:
Resolved in 860f7101405d68474e1acadf652ef640f3e43b2b.